app/browser/api/ledger.js

```diff
@@ -1408,6 +1408,54 @@ const clientprep = () => {
   _internal.verboseP = ledgerClient.prototype.boolion(process.env.LEDGER_PUBLISHER_VERBOSE)
 }
 
+/**
+ * Round trip for fetching data (scrap data from html) inside window process
+ * @param {object} params - contains params from roundtrip
+ * @param {string} params.url - url of the site that we want to scrap
+ */
+const roundTripFromWindow = (params, callback) => {
```

Comment:
please document what the arguments to `callback` are supposed to be, either here or in `roundtrip`

Reply:
done